tools/template_based_query_generation/src/test/java/graph/MarkovChainTest.java

```diff
@@ -64,7 +62,7 @@ public void test_randomWalk_anEdge() {
 	 * test for the randomWalk method when the graph consists of a bidirectional edge
 	 * an infinite loop happens in the graph.MarkovChain class, so we expect an OutOfMemoryError.class
 	 */
-	@Test
+	//@Test
```

Comment:
Is this intentional? If it is, please include some comments for it. Thx!

Reply:
Yes, will add the unit tests today!